sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -5,36 +5,39 @@
 import datetime
 import os
 import subprocess
-from typing import List, Tuple, Union
+from typing import List, Optional, Tuple, Union
 
 import numpy as np
 import pandas as pd
 import ray
 
 from sky.adaptors import aws
 
-# Turn off the regions disabled for a new AWS account by default.
+# Enable all the regions. The regions enabled by the user's AWS account will be
+# filtered by joining with the availability zone mapping in `aws_catalog`.
```

Comment:
```suggestion
# Enable most of the regions. Each user's account may have a subset of these enabled; this is ok because we take the intersection of the list here with the user-specific enabled regions in `aws_catalog`, via the "availability zone mapping".
```